src/components/Menu.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useMemo } from 'react';
+import ComboBox from './ComboBox';
+import MenuItem from './MenuItem';
+
+export const Menu = () => {
+
+   const options = useMemo(() => {
```

Comment:
You don't need `useMemo` here, it does not optimize anything. We can discuss it in the scope of this task or when we get to the hooks chapter. For now just delete it.